src/ui/pages/Settings/Settings.tsx

```diff
@@ -58,7 +61,17 @@ function SettingsMain() {
     mutationFn: () => accountPublicRPCPort.request('logout'),
   });
 
+  const {
+    data: referralProgramEnabled,
+    isLoading: isLoadingRemoteConfigValue,
+  } = useRemoteConfigValue('extension_referral_program');
+
   useBackgroundKind({ kind: 'white' });
+
+  if (isLoadingRemoteConfigValue) {
+    return null;
+  }
```

Comment:
Does this mean user will not be able to access Settings while the firebase request is in progress? What if it takes too long?

Reply:
ah, there is a 30 sec timeout...

Reply:
updated ✅